### PR TITLE
Prepare usage of crowdin: Remove underscores from l10n

### PR DESCRIPTION
**Pull request by koppor**
This PR prepares the usage of crowdin: No more strange `_` in our translation files. With this PR we are not bound to crowdin; we can still use our current GitHub worfklow. Furthermore, crowdin pushes new translations using pull requests.

A detailed discussion of crowdin can be found at https://github.com/JabRef/jabref/pull/3470.

grafik

### CODE REVIEW

File: src/main/resources/l10n/Menu_da.properties [hunk 1]
```diff
@@ -1,140 +1,136 @@
...
-#! created/edited by Popeye version 0.55 (github.com/JabRef/popeye)
-#! encoding:UTF-8
-
-Abbreviate_journal_names_(ISO)=Forkort_tidsskriftsnavn_(ISO)
-Abbreviate_journal_names_(MEDLINE)=Forkort_tidsskriftsnavn_(MEDLINE)
-About_JabRef=Om_&JabRef
-Append_library=Tilføj_ind&hold_fra_library
-Autogenerate_BibTeX_keys=&Autogenerer_BibTeX-nøgler
-Close_library=L&uk_library
+Abbreviate\ journal\ names\ (ISO)=Forkort tidsskriftsnavn (ISO)
```
Comment (Siedlerchr) [inline src/main/resources/l10n/Menu_da.properties]: But now you replaced the underscores with backslashes. Compared to that I find underscores better

**Comment by koppor** [comment]
Doesn't work otherwise in the keys, because white spaces are treated as
delimiter between key and value.
https://de.m.wikipedia.org/wiki/Java-Properties-Datei

Using underscore was a JabRef proprietary way. Using escapes seems to be
the usual way.

Also note that all the escaping in thr value field is optional!

Am 02.12.2017 12:52 schrieb "Christoph" <notifications@github.com>:

**Comment by koppor** [comment]
- [ ] adapt script to handle new entries also with `\ `

DevCall: merge after that is fixed.

**Comment by koppor** [comment]
OK, `org.jabref.logic.l10n.LocalizationConsistencyTest#findMissingLocalizationKeys` now works also with the value vield:

```
java.lang.AssertionError: DETECTED LANGUAGE KEYS WHICH ARE NOT IN THE ENGLISH LANGUAGE FILE
1. PASTE THESE INTO THE ENGLISH LANGUAGE FILE
2. EXECUTE: gradlew localizationUpdate
[
No\ more\ escaped\ characters\ in\ the\ value\ field=No more escaped characters in the value field
] 
Expected :[]
```

**Pull request merged**